spec/routing/v1/doorkeeper/authorize_routing_spec.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+RSpec.describe 'routing', type: :routing do
+  it 'routes to #new' do
+    expect(get: '/v1/oauth/authorize').to route_to('v1/doorkeeper/authorizations#new')
+  end
+
+  it 'routes to #show' do
+    expect(get: '/v1/oauth/authorize/native').to route_to('v1/doorkeeper/authorizations#show')
+  end
+
+  it 'routes to #destroy' do
+    expect(delete: '/v1/oauth/authorize/').to route_to('v1/doorkeeper/authorizations#destroy')
```

Comment:
I don't think you need the trailing slash on `/v1/oauth/authorize/`. `/v1/oauth/authorize` would be more consistent with the other paths.